### PR TITLE
BOM release doc: git diff command before starting a BOM release

### PR DESCRIPTION
**Pull request by suztomo**
I missed the google-cloud-bom bump today and the same thing happened to me before. I think I need to remind future me to confirm the changes before performing the release.

### CODE REVIEW

File: boms/cloud-oss-bom/RELEASING.md [hunk 1]
```diff
@@ -14,6 +14,31 @@ tool if you not previously done so.
...
+```
+
+If the difference includes the google-cloud-bom version, then check the change in the release note
+at https://github.com/googleapis/java-cloud-bom/releases as well.
+
+From these changes in the content of the Libraries BOM,
+determine the release version by the following logic:
+
+- If there are at least one major version bumps among the changes, it's a major version bump.
+- If there are at least one minor version bumps (no major version change), it's a minor version
```
Comment (elharo) [inline boms/cloud-oss-bom/RELEASING.md:35]: there is at least one minor version bump

File: boms/cloud-oss-bom/RELEASING.md [hunk 2]
```diff
@@ -14,6 +14,31 @@ tool if you not previously done so.
...
+git diff v16.4.0-bom -- boms/cloud-oss-bom/pom.xml
+```
+
+If the difference includes the google-cloud-bom version, then check the change in the release note
+at https://github.com/googleapis/java-cloud-bom/releases as well.
+
+From these changes in the content of the Libraries BOM,
+determine the release version by the following logic:
+
+- If there are at least one major version bumps among the changes, it's a major version bump.
```
Comment (elharo) [inline boms/cloud-oss-bom/RELEASING.md:34]: at least one major version bump

Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md]: Fixed. Thanks.

File: boms/cloud-oss-bom/RELEASING.md (hunk 1 above)
Comment (suztomo) [reply boms/cloud-oss-bom/RELEASING.md]: Fixed.

**Review by suztomo — COMMENTED** [review]
PTAL

**Review by elharo — APPROVED** [review]

**Pull request merged**